package-lock.json

```diff
@@ -5,14 +5,14 @@
   "requires": true,
   "dependencies": {
```

Comment:
@AnnaSyme can you remove this change?

Reply:
thanks @bgruening  - do I just click delete for this file (and the next one) ?